localstack/services/awslambda/usage.py

```diff
@@ -0,0 +1,10 @@
+"""
+Usage reporting for Lambda service
+"""
+from localstack.utils.analytics.usage import UsageCounter, UsageSetCounter
+
+# usage of lambda hot-reload feature
```

Comment:
nit: description could be more specific (e.g., number of functions created with hot-reload feature)